### PR TITLE
Add "Homemade Manuscript OCR (1): OCRopy"

### PR DESCRIPTION
**Pull request by jbaiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,8 @@ Contributions are welcome, as is feedback.
 * [Tutorial: Command-line OCR on a Mac](http://benschmidt.org/dighist13/?page_id=129) [@bmschmidt](https://github.com/bmschmidt)
   * Tutorial on how to run tesseract in Mac OSX
 * [Practical Expercience with OCRopus Model Training](https://comsys.informatik.uni-kiel.de/lang/de/res/practical-expercience-with-ocropus-model-training/) (2016) [@jze](https://github.com/jze)
+* [Homemade Manuscript OCR (1): OCRopy](http://graal.hypotheses.org/786) (2017) [@Jean-Baptiste-Camps](https://github.com/Jean-Baptiste-Camps)
+  * Tutorial on applying OCR to medieval manuscripts with OCRopy
```
Comment (zuphilip) [inline README.md]: Is this a subtitle, some explanation, or another blog post w/o link?

Comment (jbaiter) [inline README.md]: This is a brief explanation of what's in the Blog post, I thought I'd keep it in line with the other entries. Should I use this as the link text instead and skip the original title?

Comment (zuphilip) [inline README.md]: Ah, sorry, my fault. You are right. This looks the same as the other entries some lines above. 👍

**Comment by kba** [comment]
Thank you!

**Pull request merged**